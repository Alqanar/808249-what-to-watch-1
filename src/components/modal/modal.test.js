import React from "react";
import {mount} from "enzyme";

import Modal from "./modal.tsx";


const Child = () => <div>Yolo</div>;

let component;

const root = global.document.createElement(`div`);
root.setAttribute(`id`, `root`);
const body = global.document.querySelector(`body`);
body.appendChild(root);

afterEach(() => {
  component.unmount();
});

it(`should render all the styled components and the children`, () => {
  component = mount(
    <Modal>
      <Child />
    </Modal>,
  );
});
