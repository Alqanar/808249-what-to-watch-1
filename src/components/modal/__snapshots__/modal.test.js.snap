// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render all the styled components and the children 1`] = `
<div
  className="kek"
>
  Yolo
</div>
`;
